articles/quickstart/native/ios-objc/03-user-sessions.md

```diff
@@ -76,7 +75,7 @@ HybridAuth *auth = [[HybridAuth alloc] init];
 }];
 ```
 
-You will want to store the `accessToken` value, which is inside the `Credentials` instance. To do so, you will use an `A0SimpleKeychain` instance:
+You need a valid access token. You can find the token in the `credentials` object. To save the access token, use an `A0SimpleKeychain` instance. `SimpleKeychain` can be a key-value storage.
```

Comment:
You can find the token -> You can find the access token

Reply:
It's mentioned in the previous sentence and this way we're avoiding repetition.